x-pack/legacy/plugins/index_management/public/app/components/template_form/steps/step_mappings.tsx

```diff
@@ -67,20 +67,26 @@ export const StepMappings: React.FunctionComponent<StepProps> = ({
         </EuiFlexItem>
 
         <EuiFlexItem grow={false}>
-          <EuiButtonEmpty
-            size="s"
-            flush="right"
-            href={documentationService.getMappingDocumentationLink()}
-            target="_blank"
-            iconType="help"
-          >
-            <FormattedMessage
-              id="xpack.idxMgmt.templateForm.stepMappings.docsButtonLabel"
-              defaultMessage="Mapping docs"
-            />
-          </EuiButtonEmpty>
-          <br />
-          <LoadMappingsFromJsonButton onJson={onJsonLoaded} />
+          <EuiFlexGroup>
```

Comment:
Nit: I think we should add `gutterSize="s"` here to reduce the space between the links a bit, so they appear more as a single group.

![image](https://user-images.githubusercontent.com/1238659/72280604-90bc4400-35ed-11ea-80aa-19491578a0c8.png)